src/server_manager/web_app/app.ts

```diff
@@ -620,8 +621,13 @@ export class App {
       return;
     }
 
-    await this.loadGcpAccount(gcpAccount);
-    this.showIntro();
+    const gcpServers = await this.loadGcpAccount(gcpAccount);
+    if (gcpServers.length > 1) {
+      this.showServer(gcpServers[0]);
+    } else {
+      this.appRoot.getAndShowGcpCreateServerApp().start(this.gcpAccount);
+    }
+    bringToFront();
```

Comment:
This can be removed, I believe. The user already has the screen in the foreground.

Reply:
I don't think that's true.  The user is looking at the blank redirect target at the end of the OAuth flow, as a result of calling `this.runGcpOauthFlow()`.

Reply:
Let's move this to right after the OAuth.

Reply:
Done